src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -33,3 +33,33 @@ package util.control
  *  @author Miles Sabin
  */
 trait ControlThrowable extends Throwable with NoStackTrace
+
+object ControlThrowable {
+  sealed trait CacheableBase extends ControlThrowable {
```

Comment:
This name doesn't really reflect what it does though.

Reply:
The base trait is mainly there to reduce code duplication, and should actually be private if possible (I don't know if the compiler will allow it)